src/GitHub.Api/Helpers/AssemblyResources.cs

```diff
@@ -33,7 +33,22 @@ public static NPath ToFile(ResourceType resourceType, string resource, NPath des
             if (stream != null)
                 return destinationPath.Combine(resource).WriteAllBytes(stream.ToByteArray());
 
-            return environment.ExtensionInstallPath.Combine(type, os, resource).Copy(destinationPath.Combine(resource));
+            // check the GitHub.Api assembly
```

Comment:
I've looked at this method before and I've always wondered about some of the paths it takes and returns from.
Can you tell me about this function so I can add comments to it?